src/components/MultiSelect.tsx

```diff
@@ -11,7 +11,12 @@ const MultiSelect = (props: MultiSelectProperties): React.JSX.Element => {
   const [listWidth, setListWidth] = useState<string | number>(props.listStyle?.width ?? 0);
   const [listX, setListX] = useState<number>(0);
   const [listDisplay, setListDisplay] = useState<boolean>(false);
-  const [pos, setPos] = useState<SelectorPos>({top: 0, bottom: 0});
+  const [refRect, setRefRect] = useState<SelectorRect>({
+    x: 0,
+    y: 0,
+    width: 0,
+    height: 0,
+  });
```

Comment:
Why are we keeping `listX` and `listWidth` if both values are not part of `SelectorRect`? Also, both `set` methods are never used, so they should probably just be `const` variables if we keep them

Reply:
They were supposed to be removed, but a merge at some point probably inserted them and I haven't noticed. I'll remove them.